### DISCUSSION TITLE
Icon request: Ladder

### DISCUSSION THREAD

**Discussion by pederdin** [FortAwesome/Font-Awesome#21176] [Icon Requests]
Hi

I am converting to Fontawesome in several projects. Now i ran into an icon that I need: A ladder.

We are building a sports community where anyone can set up a sports ladder and would love to have an icon for this cause.

Thanks for great work.

/Peder

**Comment by ChickenSniper** [comment]
+1

**Comment by pabermod** [comment]
+1

**Comment by sarahtotally** [comment]
+1 we need a latter for a "Career Ladder" page

**Comment by ZachThomasAU** [comment]
+1

**Comment by jeplx** [comment]
+1

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
🪜 [U+1FA9C](https://emojipedia.org/emoji/%F0%9F%AA%9C/) Ladder

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by brentcrammond** [comment]
+1

**Comment by annelievrielink** [comment]
<img width="162" height="186" alt="stairs" src="https://github.com/user-attachments/assets/190efdd7-3b69-4e89-b3d9-9823c92e0830" />
Yes please, we also need this to convert to Font awsome completly
